### ISSUE TITLE
Twistd "basics" guide references non-existent Debian archive

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3635]
|<img alt="jparise's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| jparise reported|
|-|-|
|Trac ID|trac#3635|
|Type|defect|
|Created|2009-01-28 07:29:48Z|
|Branch|https://github.com/twisted/twisted/tree/twistd-debian-archive-3635|

`howto/basics.html` references a "beta Debian archive" maintained by Moshe that no longer appears to exist.  I suggest simply removing that reference from the documentation.

Attachments:

* [basics.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/0ad/0ad0d0093a5e13d5643433c5623f3504286b1a44/ecad3d7a4ad4f824821d3bbc37dce25f7ed2aea7.patch) (470 bytes) - added by jparise on 2009-01-28 07:30:12Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__3635 3635
type__defect defect
reporter__jparise jparise
priority__low low
milestone__ 
branch__branches_twistd_debian_archive_3635 branches/twistd-debian-archive-3635
branch_author__jparise__thijs jparise, thijs
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1233127788000000 1233127788000000
changetime__1237750406000000 1237750406000000
version__None None
owner__ 
cc__thijs cc__exarkun
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

What's going on with this ticket?

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

(In [26508]) Branching to 'twistd-debian-archive-3635'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra commented|
|-|-|

(In [26509]) Apply basics.patch: removing reference to beta Debian archive that no longer appears to exist (refs [#3635](https://github.com/twisted/twisted/issues/3635))

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

Putting it up for review, [doc builder](http://buildbot.twistedmatrix.com/builders/documentation/builds/361) is happy.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

Please merge

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_7"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [26512]) Merge twistd-debian-archive-3635: removing reference to beta Debian archive that no longer appears to exist

Author: jparise, thijs
Reviewer: exarkun
Fixes: [#3635](https://github.com/twisted/twisted/issues/3635)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
